packages/race-cli/src/reporters/html-reporter.ts

```diff
@@ -15,16 +15,16 @@ export class HtmlReporter extends BaseRacepointReporter {
     this._reportPath = outputTarget || '.';
   }
 
-  process = (results: LighthouseResultsWrapper): undefined => {
+  process = (results: UserFlowResultsWrapper): undefined => {
     const htmlPath = `${this._reportPath}/${formatFilename({
-      url: results.lhr.requestedUrl,
-      date: results.lhr.fetchTime,
+      url: results.steps[0].lhr.requestedUrl,
```

Comment:
is the html reporter always going to be concerned with only the first step, or will there be others?

Reply:
Its only looking for the URL which is the same on all reports